MidasCivil_Adapter/AdapterActions/Execute.cs

```diff
@@ -44,64 +46,110 @@ public override Output<List<object>, bool> Execute(IExecuteCommand command, Acti
         {
             var output = new Output<List<object>, bool>() { Item1 = null, Item2 = false };
 
-            output.Item2 = RunCommand(command as dynamic);
+            output.Item2 = RunCommandWithTaskCompletionSource(command as dynamic);
```

Comment:
```suggestion
            output.Item2 = RunCommandWithTaskCompletionSource(command);
```
Don't think this needs to be dynamic, as you're just passing it to `RunCommandWithTaskCompletionSource`.